dataservice/api/diagnosis/schemas.py

```diff
@@ -0,0 +1,36 @@
+from dataservice.api.diagnosis.models import Diagnosis
+from dataservice.api.common.schemas import BaseSchema
+from marshmallow_sqlalchemy import field_for
+from marshmallow import validates, ValidationError
+from dataservice.extensions import ma
+
+MIN_AGE_DAYS = 0
+MAX_AGE_DAYS = 32872
```

Comment:
Isn't `INTEGER` type 4 bytes?

Reply:
Yes, but I set the max based on the kids first max for age_at_diagnosis_days which is roughly 90 years.

Reply:
where is this defined?

Reply:
https://github.com/kids-first/kf-dictionary/blob/72aab588f06469ab74abaf406e52ef2d35e50289/gdcdictionary/schemas/diagnosis.yaml#L45

Reply:
hmm ok. But I think we can ignore that because we'll probably be taking that field out of the kf dictionary.

Reply:
👍 I'll change the max to something that makes more sense for us (i.e. maybe 125 years or something like that?). @allisonheath - what do you think a reasonable max should be for age_at_event_days?

Reply:
HIPAA safe harbor guidelines is that everyone above the age of 90 should be considered one group: https://www.hhs.gov/hipaa/for-professionals/privacy/special-topics/de-identification/index.html#standard

This is where 32872 (90 years in days) comes from and we should definitely keep it to ensure deidentification.

Reply:
ooh forgot about that one 👍

Reply:
👍